test/WideImage/Mapper/BMPTest.php

```diff
@@ -36,12 +36,12 @@ class BMPTest extends WideImage_TestCase
      */
     protected $mapper;
 
-    public function setup()
+    public function doSetUp()
```

Comment:
I would prefer us not to get too reliant on the symfony bridge, but rather get rid of that in v3 with higher requirements. Instead this can be renamed and `@before` / `@after` can be used to trigger the desired behaviour.

Check https://github.com/smottt/WideImage/commit/d793a568fd8bfc6f5527a079900120022efeab95#diff-e45ef9c06329f6f6cc12dd3f9c804c932dc52f81148c3914ee4ffdec329703faR37 for example.